server/schedulers/evict_leader.go

```diff
@@ -55,11 +55,11 @@ func init() {
 
 			id, err := strconv.ParseUint(args[0], 10, 64)
 			if err != nil {
-				return errors.WithStack(err)
+				return errs.ErrSchedulerConfig.Wrap(err).FastGenByArgs("id")
```

Comment:
Will it change the output of API?

Reply:
I am not sure whether the wrapped error will affect the outside